README.md

```diff
@@ -102,6 +102,9 @@ name: Publish Allure report to Morph object storage
 on:
   push:
     branches: [ main ]
+
+permissions: write-all
```

Comment:
I doubt it's a good choice for example. People will copy/paste it and this setting is very wide open. Why do you need write access, for example? Same question applies to our action test.

Reply:
> I doubt it's a good choice for example. People will copy/paste it and this setting is very wide open. Why do you need write access, for example? Same question applies to our action test.

Fixed